qcodes/config/qcodesrc_schema.json

```diff
@@ -24,6 +24,18 @@
                         "DEBUG"
                     ]
                 },
+                "file_loglevel" :{
+                    "type" : "string",
+                    "description": "control file logging  level",
```

Comment:
Just to be super nitpick there is an extra space here

Reply:
That is a reminder of the incredible fast coding from Giulio ;-). I just checked, it has been copy and pasted to almost every entry of the config, because nobody just writes jason by hand....

Reply:
:) Who is this Jason and does he have the a golden fleece